.github/workflows/rw-backup-auzre-maria-database.yml

```diff
@@ -70,6 +70,9 @@ jobs:
     uses: ./.github/workflows/rw-change-azure-maria-firewall.yml
     with:
       github_env: ${{ inputs.github_env }}
+      add_or_remove: add
+      database_resource_group: ${{ inputs.database_resource_group }}
+      database_server_name: ${{ inputs.database_server_name }}
```

Comment:
You'll need to add same to `remove_db_firewall_rule` section

Reply:
add_or_remove: remove
      database_resource_group: ${{ inputs.database_resource_group }}
      database_server_name: ${{ inputs.database_server_name }}